hello-world/app/scripts/deploy_service.js

```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const oasis = require('@oasislabs/client');
 
-oasis.workspace.HelloWorld.deploy({
+oasis.workspace.HelloWorld.deploy('pikachu',  ['snorlex', 'ghastly'], {
```

Comment:
`HelloWorld` does not need any arguments, does it?

The arguments in `deploy` should match the constructor (`new`) in the service, minus the `Context`: https://github.com/oasislabs/tutorials/blob/master/hello-world/service/src/main.rs#L20

Reply:
you're right - accidentally made this change, reverted it!